Sources/Extensions/UIKit/UIRefreshControlExtensions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIRefreshControlExtensions.swift
+//  SwifterSwift
+//
+//  Created by ratul sharker on 7/24/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(UIKit)
+import UIKit
+
+#if !os(watchOS)
+// MARK: - Methods
+public extension UIRefreshControl {
+
+    /// SwifterSwift: Programatically begin refresh control inside of UITableView.
+    ///
+    /// - Parameter tableView: UITableView instance, inside which the refresh control is contained.
+    public func beginRefreshProgramatically(in tableView: UITableView) {
+        //https://stackoverflow.com/questions/14718850/uirefreshcontrol-beginrefreshing-not-working-when-uitableviewcontroller-is-ins/14719658#14719658
+        beginRefreshing()
+        let offsetPoint = CGPoint.init(x: 0, y: -frame.size.height)
```

Comment:
If you noticed in my examples above, you can shorten this by removing `.init` and `.size`.

Reply:
Before seeing this comment, i didn't know that the `CGRect` has a computed property name `height`. Today i learned it from your comment. Thanks a lot @guykogus 

I also minimized one test case as follows:
```
XCTAssertEqual(tableView.contentOffset.y, -refreshControl.frame.height)
```